src/JobRunner.php

```diff
@@ -42,12 +42,12 @@ public function __construct(
 	/**
 	 * Adds a job to the JobRunner instance.
 	 *
-	 * @param string $class      Path to the Job class.
-	 * @param array  $definition Job definition (e.g. interval).
+	 * @param string         $class      Path to the Job class.
+	 * @param JobDefinition  $definition Job definition (e.g. interval).
 	 * @throws InvalidArgumentException If the given class does not subclass Job.
 	 * @return void
 	 */
-	public function addJob($class, array $definition = array())
+	public function addJob($class, JobDefinition $definition = null)
```

Comment:
If you contain the class in `JobDefinition`, this would only take a single param.

`$job_runner->addJob(new JobDefinition(MyJob::class, true, null, 3600));`